### DISCUSSION TITLE
Use for `complete-symbol`?

### DISCUSSION THREAD

**Discussion by yatesco** [minad/vertico#406] [Q&A] [answered]
Hi - I've installed this and it's working in `M-x` for example, but when I call `C-M-i` (`complete-symbol`) I see see a non-interactive list in a new buffer called `*Completions*`.

Am I misunderstanding this? I thought this would allow me to achieve effectively what `corfu` does?

Thanks!

**Comment by minad** [comment, marked as answer]
You are looking for `consult-completion-in-region`, see https://github.com/minad/vertico#completion-at-point-and-completion-in-region.

**Reply by yatesco** [reply]
thanks!
